pennylane_lightning/src/benchmarks/CMakeLists.txt

```diff
@@ -46,8 +46,8 @@ endif()
 
 add_library(lightning_benchmarks_dependency INTERFACE)
 target_link_libraries(lightning_benchmarks_dependency INTERFACE lightning_utils
-                                                    lightning_external_libs
-                                                    benchmark::benchmark_main)
+                                                                lightning_external_libs
+                                                                benchmark::benchmark)
```

Comment:
Should this be
```suggestion
                                                                benchmark::benchmark_main)
```

Reply:
As we provide a custom main for `bench_kernels` using `benchmark::benchmark` is correct. Still, I rearranged the library dependency to make CMake file clear.